sui/token_bridge/sources/complete_transfer.move

```diff
@@ -84,11 +110,165 @@ module token_bridge::complete_transfer {
                 ctx
             );
         };
-
         // take out fee from the recipient's coins. `extract` will revert
         // if fee > amount
         let fee_coins = coin::split(&mut recipient_coins, fee_amount, ctx);
         transfer_object::transfer(recipient_coins, recipient);
         transfer_object::transfer(fee_coins, fee_recipient);
     }
 }
+
+#[test_only]
+module token_bridge::test_complete_transfer {
```

Comment:
in most other places we use the `complete_transfer_test` naming convention, though I'm now seeing a couple exceptions to that rule, which we should fix up too (low priority)

Reply:
I think in most places it is `module token_bridge::test_something` rather than of something_test. Which do you think is better?

Reply:
We should just stick to whatever is used more in the rest of the codebase (probably `..._test`) although I personally like (`test_...`) because I can tell that its a test faster.

Reply:
a quick grep in the sui folder:
```
module token_bridge::test_register_chain {
module wormhole::deserialize_test {
module wormhole::external_address_test {
module wormhole::test_state{
module token_bridge::string32_test {
module token_bridge::normalized_amount_test {
// module wormhole::emitter_test {
module wormhole::vaa_test {
module wormhole::test_serialize {
module wormhole::guardian_pubkey_test {
module wormhole::guardian_set_upgrade_test {
module token_bridge::test_bridge_state{
module token_bridge::token_bridge_vaa_test{
module token_bridge::transfer_test {
```

which means 10 `*_test` and 4 `test_*`.